html/index.html

```diff
@@ -0,0 +1,74 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>TASK 3</title>
+        <link rel="stylesheet" href="../css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="headerstarts">
+                <header>
+                    <img src="../images/logo1.png" alt="logo"> 
+                    <h1>DONVICO COLLECTIONS</h1>
+                    <nav>
+                        <ul>
+                            <li><a href="./index.html">HOME</a></li>
+                            <li><a href="./about.html">ABOUT US</a></li>
+                            <li><a href="./contact.html">CONTACT</a></li>
+                        </ul>
+                    </nav>
+                </header>
+            </div>
+            <section>
+                <article>
```

Comment:
Kindly fix this aspect
You are expected to create SEVERAL articles with EACH containing a h2 and paragraph of text